cfgov/v1/models/learn_page.py

```diff
@@ -368,6 +368,19 @@ def event_state(self):
 
         return "future"
 
+    @cached_property
+    def location_str(self):
+        parts = []
+
+        if self.venue_city and self.venue_state:
+            parts.append(f"{self.venue_city}, {self.venue_state}")
```

Comment:
I think we need to add a comma after `{self.venue_state}` IF any other values appear in the `location_str`.

Reply:
I agree that would be better but I was trying to keep this consistent with the existing implementation, which I thought had been updated recently. But apparently this logic [is quite old](https://github.com/cfpb/consumerfinance.gov/commit/de9cbd04cf19bef447c150409293edaa532a88c6), so I will update.

Reply:
@anselmbradford updated in latest commit. I chose to also add a comma before "Livecast" so we do

> Washington, DC, Livecast
> CFPB HQ, Livecast
> Washington DC, CFPB HQ, Livecast

instead of 

> Washington, DC Livecast
> CFPB HQ, Livecast
> Washington, DC, CFPB HQ Livecast